### ISSUE TITLE
prerelease mangles UTF-8 in the changelog

### ISSUE THREAD

**Issue by mgedmin** [zestsoftware/zest.releaser#254]
Steps to reproduce:

- git clone https://github.com/mgedmin/objgraph && cd objgraph
- prerelease
- git diff

Expected result

- release date is changed, nothing else is

Actual result

- "Kristján" is mangled into "KristjÃ¡n"

**Comment by reinout** [comment]
When I try it, I get this `git diff` output:

    diff --git a/CHANGES.rst b/CHANGES.rst
    index caab284..b803dc5 100644
    --- a/CHANGES.rst
    +++ b/CHANGES.rst
    @@ -3,7 +3,7 @@ Changes

     .. currentmodule:: objgraph

    -3.1.2 (unreleased)
    +3.1.2 (2017-11-24)
     ------------------

     - Correct UTF-8 mojibake in the changelog and switch all links to HTTPS.
    diff --git a/objgraph.py b/objgraph.py
    index 22fdd20..f12db95 100755
    --- a/objgraph.py
    +++ b/objgraph.py
    @@ -54,7 +54,7 @@ except ImportError:
     __author__ = "Marius Gedminas (marius@gedmin.as)"
     __copyright__ = "Copyright (c) 2008-2017 Marius Gedminas and contributors"
     __license__ = "MIT"
    -__version__ = '3.1.2.dev0'
    +__version__ = '3.1.2'
     __date__ = "2017-10-30"

So: looks fine. I've installed zest.releaser with python 2.7 and I'm running it on OSX. Are you running it with python 3, for instance?

There's some code in zest.releaser that tries to keep everything working even with accented characters, though this is of course a difficult area.

**Comment by mgedmin** [comment]
I'm running on Python 2.7.   I did a `pip install -U zest.releaser` before trying to reproduce and filing this bug.

**Comment by mgedmin** [comment]
Hm, if I run `prerelease -v`, I see

```
...
DEBUG: Checking CHANGES.rst
...
DEBUG: utf-8  confidence = 0.505
...
DEBUG: Detected encoding of CHANGES.rst with chardet: ISO-8859-1
...
```

which could be relevant?

FWIW my locale is

```
LANG=lt_LT.UTF-8
LANGUAGE=
LC_CTYPE=lt_LT.UTF-8
LC_NUMERIC="lt_LT.UTF-8"
LC_TIME="lt_LT.UTF-8"
LC_COLLATE="lt_LT.UTF-8"
LC_MONETARY="lt_LT.UTF-8"
LC_MESSAGES="lt_LT.UTF-8"
LC_PAPER="lt_LT.UTF-8"
LC_NAME="lt_LT.UTF-8"
LC_ADDRESS="lt_LT.UTF-8"
LC_TELEPHONE="lt_LT.UTF-8"
LC_MEASUREMENT="lt_LT.UTF-8"
LC_IDENTIFICATION="lt_LT.UTF-8"
LC_ALL=
```

**Comment by mgedmin** [comment]
Oh my, see #255 for my theory of why this is happening.

**Comment by reinout** [comment]
Thanks for #255, I've released 6.13.2 with your fix!
